template/workflows/manifests/.github/workflows/azure-kubernetes-service.yml

```diff
@@ -95,8 +97,17 @@ jobs:
         with:
           kubelogin-version: 'v0.0.25'
 
+      # gets credentials for a fleet
+      - name: Get fleet credentials
+        if: env.FLEET == 'true'
```

Comment:
Not sure if this is possible, but can we instead check whether env.FLEET_NAME is empty or not?

Reply:
This comment applies to all `env.FLEET == 'true'` expressions below as well.